includes/admin/builder/class-evf-builder-settings.php

```diff
@@ -272,11 +273,9 @@ public function output_content() {
 					$this->form_data,
 					__( 'To Address', 'everest-forms' ),
 					array(
-						'default' => isset( $settings['email'][$connection_id]['evf_to_email'] ) ? $settings['email'][$connection_id]['evf_to_email'] : get_option( 'admin_email' ),
-						'smarttags'  => array(
-							'type'   => 'fields',
-							'form_fields' => 'email',
-						),
+						'default'    => '{admin_email}',
+						'tooltip'    => __('Enter the Email address where you want to send the email.', 'everest_forms'),
```

Comment:
Something like this will be best helpful information for user because tooltip are for user not for us:
- Rather than "you want to send the email", say something like "where you want to receive the entry email notification" which I think is cool sentence.
- Since multiple-email entry in this field is supported so also include something like "For multiple notifications, separate email addresses with a comma."..... Because many user may be in dilema about this multiple email support

Reply:
Fixed :)